docs/src/Box.doc.js

```diff
@@ -645,6 +645,46 @@ card(
     <Text color="white">"fit" sets width to 100% </Text>
   </Box>
 </Box>
+`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      description={`When content overflows the bounds of the Box, there are multiple options to control the overflow behavior. The default is \`overflow="visible"\`.`}
```

Comment:
no article

Reply:
I'm happy to go with this as is. 

It feels a little cluttered to me. We have lots of text, behavior, and color. 

I think we could go with 1 combo of colors.
Then each prop type, if needed, explained in the description rather than inside each Box
Then Box example be just: 

```
overflow="hidden"

_Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed vitae nisl nec turpis vehicula ultrices. Duis pretium ut ipsum nec interdum. Vestibulum arcu dolor, consectetur ac eros a, varius commodo justo. Maecenas tincidunt neque elit, eu pretium arcu dictum ac. Donec vehicula mauris ut erat dictum, eget tempus elit luctus. In volutpat felis justo, et venenatis arcu viverra in. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Proin enim lorem, vulputate eget imperdiet nec, dapibus sed diam. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Suspendisse rhoncus ut leo non gravida. Nulla tincidunt tellus sit amet ornare venenatis. Sed quis lorem cursus, porttitor tellus sed, commodo ex. Praesent blandit pretium faucibus. Aenean orci tellus, vulputate id sapien sit amet, porta fermentum quam. Praesent sem risus, tristique sit amet pulvinar in, scelerisque sit amet massa._
```

Reply:
Great call, updated!